sound/soc/sof/loader.c

```diff
@@ -192,6 +192,20 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				    const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	int i;
+
+	/* TODO: Currently HPRO/LPRO only used. Other data will be parsed later
+	 * for clk_config, 0:HPRO, x:LPRO
+	 */
+	sdev->clk_config = config_data->elems[SOF_EXT_MAN_CONFIG_CAVS_LPRO].value;
```

Comment:
@fredoh9 looks likethe config_data can have zero or more elems isnt it? Dont you have to find the number of elems and then parse depending on it?

Reply:
you need to iterate through the elems, check the type and then store the value isnt it?

Reply:
My understanding from https://github.com/thesofproject/sof/pull/2893, It didn't seem to have multiple of same element. And also I don't see any reason to have multiple of same element.

Reply:
My bad, @ranj063 is right. I will update again.